src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -342,10 +351,14 @@ export default stripesFinalForm({
 
       tools.changeValue(state, 'records', () => records);
     },
-    restoreRecord: ([{ index, record }], state, tools) => {
-      const records = restoreRecordAtIndex(index, record, state);
+    restoreRecord: ([{ index }], state, tools) => {
+      const records = restoreRecordAtIndex(index, state);
 
       tools.changeValue(state, 'records', () => records);
     },
   },
+  subscription: {
+    dirty: true,
+    dirtyFields: true,
+  },
```

Comment:
Are these lines necessary if we use isEqual in FieldArray?

Reply:
Oh, I used that for debugging and forgot to remove, thanks